File: LeetCode/java/1848_Minimum_Distance_to_the_Target_Element.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        List<Integer> indexes = new ArrayList<>();
        int answer = nums.length;

        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == target) {
                indexes.add(i);
            }
        }

        for (int index : indexes) {
            answer = Math.min(answer, Math.abs(index - start));
        }

        return answer;
    }
}

File: LeetCode/cpp/1848_Minimum_Distance_to_the_Target_Element.cpp
class Solution {
public:
	int getMinDistance(vector<int>& nums, int target, int start) {
		vector<int> indexes;
		int answer = nums.size();

		for (int i = 0; i < nums.size(); i++) {
			if (nums[i] == target) {
				indexes.push_back(i);
			}
		}

		for (int index : indexes) {
			answer = min(answer, abs(index - start));
		}

		return answer;
	}
};